build/syntax.go

```diff
@@ -314,7 +319,12 @@ type TupleExpr struct {
 }
 
 func (x *TupleExpr) Span() (start, end Position) {
-	return x.Start, x.End.Pos.add(")")
+	if x.Start.IsValid() {
+		return x.Start, x.End.Pos.add(")")
+	}
+	start, _ = x.List[0].Span()
+	_, end = x.List[len(x.List)-1].Span()
+	return
```

Comment:
nit: I always prefer 'return start, end' so that you don't have to look at the signature to figure out what variables were in the return

Reply:
Done.